src/modules/rangeproof/tests_impl.h

```diff
@@ -463,6 +464,30 @@ static void test_single_value_proof(uint64_t val) {
         CHECK(plen == 73);
     }
 
+    /* Test if trailing bytes are rejected. */
+    proof[plen] = 0;
+    CHECK(secp256k1_rangeproof_verify(
+        CTX,
+        &min_val_out, &max_val_out,
+        &commit,
+        proof, plen + 1,
```

Comment:
`proof` should probably be fully initialized to avoid a read of uninitialized data here. (I think `secp256k1_rangeproof_verify` will not read the last byte, but the test shouldn't rely on that.)

Reply:
I do initialize the last byte in the line above this check.

Reply:
oh, nevermind